src/components/cc-email-list/cc-email-list.smart.js

```diff
@@ -1,9 +1,6 @@
-import {
-  todo_addEmailAddress as addEmailAddress,
-  todo_getEmailAddresses as getEmailAddresses,
-  todo_removeEmailAddress as removeEmailAddress,
-  todo_getConfirmationEmail as sendConfirmationEmail,
-} from '@clevercloud/client/esm/api/v2/user.js';
+// prettier-ignore
```

Comment:
question: why prettier-ignore ?

Reply:
:joy: brace yourself, here comes the explanation:
- I have an import from `clever-client.js`,
- I want to add a `@ts-expect-error` because it does not come with types,
- The `@ts-expect-error` must always be added right on top of the error you want to silence,
- So when the import spans through several lines, I would like the `@ts-expect-error` to be just above `from '...';`,
- When I add the comment, `prettier` removes it,
- If I try to `// prettier-ignore` and then `@ts-expect-error` within a multi-line import, `prettier` removes both comments (I guess no comment is allowed within imports whatsoever).

=> Hence I force the import to be single-line + I add `prettier-ignore` so that `prettier` does not make it multi-line and then I add `@ts-expect-error` :man-shrugging: